File: spring/src/main/java/com/enzo/testemuralis/dto/ContatoResponseDTO.java
package com.enzo.testemuralis.dto;

import com.enzo.testemuralis.models.Contato;

public record ContatoResponseDTO(
    String tipo,
    String texto
) {
    public ContatoResponseDTO(Contato contato) {
        this(contato.getTipo(), contato.getTexto());
    }
}

File: dotnet/TesteMuralis/Dto/ContatoResponseDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using TesteMuralis.Models;

namespace TesteMuralis.Dto
{
    public class ContatoResponseDTO
    {
        public string Tipo { get; set; }
        public string Texto { get; set; }

        public ContatoResponseDTO(Contato contato)
        {
            Tipo = contato.Tipo;
            Texto = contato.Texto;
        }
    }
}